src/core/stream_send.c

```diff
@@ -1508,11 +1593,40 @@ QuicStreamOnResetAck(
             "[strm][%p] Send State: %hhu",
             Stream,
             QuicStreamSendGetState(Stream));
+        QuicTraceLogStreamVerbose(
+            OnResetAck,
+            Stream,
+            "Shutting down stream [abortively] from OnResetAck. Send Side.");
         QuicStreamIndicateSendShutdownComplete(Stream, FALSE);
         QuicStreamTryCompleteShutdown(Stream);
     }
 }
 
+_IRQL_requires_max_(PASSIVE_LEVEL)
+void
+QuicStreamOnResetReliableAck(
+    _In_ QUIC_STREAM* Stream
+) {
+    Stream->Flags.LocalCloseResetReliableAcked = TRUE;
+    QuicTraceLogStreamVerbose(
+            ResetReliableAck,
+            Stream,
+            "Reset Reliable ACKed in OnResetReliableAck. Send side. UnAckedOffset=%llu, ReliableOffsetSend=%llu",
+            Stream->UnAckedOffset, Stream->ReliableOffsetSend);
+    if (Stream->UnAckedOffset >= Stream->ReliableOffsetSend && !Stream->Flags.LocalCloseAcked) {
+        Stream->Flags.LocalCloseAcked = TRUE;
+        Stream->Flags.RemoteCloseAcked = TRUE;
```

Comment:
We must log `StreamSendState` log any time that state changes. Tooling automation uses this.

Reply:
Noted. I can add StreamSendState.